runtime/doc/news.txt

```diff
@@ -86,6 +86,13 @@ EVENTS
 
 • |vim.ui_attach()| callbacks for |ui-messages| `msg_show` events are executed in
   |api-fast| context.
+• Various additions for the following UI events:
+  • `cmdline_show`: `hl_id` argument to highlight the prompt text.
+  • `cmdline_hide`: `abort` argument indicating if the cmdline was aborted.
```

Comment:
nit: for consistency, this should probably be `cancel`. In `:help api`, `:help lsp`, and `:help CompleteDone`, we use "cancel" in a few places and "abort" nowhere.

"abort" can be useful when we want to indicate a "ungraceful exit" where state may be invalid, but I guess that's not the case here.
```suggestion
  • `cmdline_hide`: `cancel` indicates the cmdline was canceled.
```

Reply:
Fair, but I think that is trumped by the fact that this is analogous to the `v:event.abort` field for `CmdlineLeave`.